lib/src/chart/radar_chart/radar_chart_data.dart

```diff
@@ -351,27 +374,30 @@ class RadarEntry with EquatableMixin {
   /// [RadarChart] draws every point or entry with [RadarEntry]
   const RadarEntry({required double value}) : value = value;
 
+  /// Used for equality check, see [EquatableMixin].
+  @override
+  List<Object?> get props => [value];
+
   /// Lerps a [RadarEntry] based on [t] value, check [Tween.lerp].
-  RadarEntry copyWith({double? value}) =>
-      RadarEntry(value: value ?? this.value);
+  RadarEntry copyWith({double? value}) => RadarEntry(value: value ?? this.value);
 
   /// Lerps a [RadarDataSet] based on [t] value, check [Tween.lerp].
   static RadarEntry lerp(RadarEntry a, RadarEntry b, double t) {
     return RadarEntry(value: lerpDouble(a.value, b.value, t)!);
   }
+}
 
-  /// Used for equality check, see [EquatableMixin].
-  @override
-  List<Object?> get props => [value];
+enum RadarShape {
```

Comment:
There is a lot of code displacement. Please revert them all.

Reply:
:maximumfacepalm: I must have formatted the code by reflex. I'm sorry, I'll revert it in the next commit...

Reply:
Thank you!